src/main/java/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
you can use record

if there's a problem with Checkstyle - update the plugin to 3.3.0